backend/api/app/domains/user/repositories.py

```diff
@@ -23,6 +23,32 @@ def get_by_id(user_id):
     def get_all():
         return User.query.all()
     
+    @staticmethod
+    def updateUser(user):
+        try:
+            db.session.add(user)  
+            db.session.commit()  
+        except Exception as e:
+            db.session.rollback()  
+            raise e
```

Comment:
Try to more precisely define these type of error, so that we don't expose our model/db/stacktrace info